_src-js/lib/bound-properties.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import ReactDOM from 'react-dom'
+import videojs from 'video.js';
+import shortid from 'shortid';
+
+const Form = React.createClass({
+  getInitialState() {
+    return {
```

Comment:
didn't realize you could do that. This is my first time using React and was just following their tutorial.
Though, totally makes sense that that should work.